spyder/plugins/editor.py

```diff
@@ -966,7 +975,7 @@ def get_plugin_actions(self):
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 
         file_menu_actions = [self.new_action,
-                             None,

```

Comment:
Please add one more space here to align this line to the bracket (`[`) in the previous line.